packages/s/stackwalker/xmake.lua

```diff
@@ -0,0 +1,34 @@
+package("stackwalker")
+
+    set_homepage("https://curl.haxx.se/")
+    set_description("The multiprotocol file transfer library.")
+
+    set_urls("https://github.com/JochenKalmbach/StackWalker/archive/$(version).zip",
+             "https://github.com/JochenKalmbach/StackWalker.git")
+    add_versions("1.20", "b139c83b7c4991930ebe48eae43b0feedca034e136f00be294f3641495b2c835")
+
+    add_deps("cmake")
+    add_syslinks("advapi32")
```

Comment:
Needs to be wrapper in windows platform check

Reply:
This is a Windows only library, is this really necessary?